patches/tinyemu/tinyemu/temu.c

```diff
@@ -591,276 +581,6 @@ static EthernetDevice *slirp_open(void)
 
 #endif /* CONFIG_SLIRP */
 
-/*******************************************************/
-/* socket */
```

Comment:
Please do not remove the original networking logic because it's needed for non-wamr runtimes (e.g. wasmtime, wazero and browser). We can enable the networking logic for wamr when `-net=wamr` is passed to the emulator flag during runtime.

Reply:
Whoopsies, no idea how that happened, must have been through the rebase. Will check and reapply customizations again.